src/react/components/toolbar/previousAsset.tsx

```diff
@@ -0,0 +1,11 @@
+import { ToolbarItem } from "./toolbarItem";
```

Comment:
likely more trouble than what it's worth, but looking at previous/next in terms of DRY, an AssetNav component that takes direction?

Reply:
Totally makes sense 👍